coremltools/converters/mil/mil/ops/defs/control_flow.py

```diff
@@ -157,29 +175,28 @@ def value_inference(self):
 @register_op(doc_str="")
 class select(Operation):
     """
-    Returns the elements selected from either ``a`` or ``b``, depending on
-    ``cond``. Shape of ``cond``, ``a``, ``b`` must be broadcastable.
-
-    ``a, b`` must be provided together, or neither is provided. If neither is
-    provided, returns the indices of ``cond`` that are ``True``.
+    Return the elements selected from either ``a`` or ``b``, depending on the ``cond``.
+    
+    The shape of ``cond``, ``a``, and ``b`` must be broadcastable. You must provide ``a, b`` together, or provide neither. If you provide neither, the operation returns the indices of ``cond`` that are ``True``.
```

Comment:
Might be better to keep the line shorter for readability? (e.g. < 100?) New lines should be ignored on the rendered page.